dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -625,16 +691,25 @@ private static void parseLineResults(Counters counters, LineImportResult lineRes
             List<String> updatedInodes, final boolean updateCounters) {
 
         if (updateCounters) {
-            counters.setContentToCreate(
-                    counters.getContentToCreate() + lineResult.contentToCreate());
-            counters.setContentCreated(
-                    counters.getContentCreated() + lineResult.createdContent());
-            counters.setContentToUpdate(
-                    counters.getContentToUpdate() + lineResult.contentToUpdate());
-            counters.setContentUpdated(
-                    counters.getContentUpdated() + lineResult.updatedContent());
-            counters.setContentUpdatedDuplicated(
-                    counters.getContentUpdatedDuplicated() + lineResult.duplicateContent());
+            //counters.setContentToCreate(counters.getContentToCreate() + lineResult.contentToCreate());
```

Comment:
I think we can remove these code comments

Reply:
done